### PR TITLE
Multiple kafka broker connections

### PR DESCRIPTION
**Pull request by stefanovik**
Hello,

Created a way to configure multiple kafka broker connections, particularly because we've run into a situation where we needed to connect to kafka brokers hosted on our own servers and kafka brokers from Amazon MSK, so we needed this duality.

### CODE REVIEW

**Review by mihaileu — APPROVED** [review]

File: src/Console/Commands/KafkaConsumerCommand.php [hunk 1]
```diff
@@ -53,6 +63,16 @@ public function handle()
 
             return;
         }
+
+        if ($this->option('brokerConnection')) {
+            $this->config['sasl'] = config(
+                'kafka.connections.' . $this->option('brokerConnection') . '.sasl'
```
Comment (mateusjunges) [inline src/Console/Commands/KafkaConsumerCommand.php:69]: Please use brace syntax here: 

```
"kafka.connections.{$this->option('brokerConnection')}.sasl"
```

File: src/Console/Commands/KafkaConsumerCommand.php [hunk 2]
```diff
@@ -28,14 +29,23 @@ public function __construct()
     {
         parent::__construct();
 
+        $securityProtocol = config('kafka.default') ? config(
+            'kafka.connections.' . config('kafka.default') . '.securityProtocol'
+        ) : config('kafka.securityProtocol');
+        $sasl = (config('kafka.default') ? config(
+            'kafka.connections.' . config('kafka.default') . '.sasl'
+        ) : config('kafka.sasl')) ?? [];
```
Comment (mateusjunges) [inline src/Console/Commands/KafkaConsumerCommand.php:37]: This isn't very readable to me

**Review by mateusjunges — CHANGES_REQUESTED** [review]
Hey! Thanks for your work on this. I have some considerations I'd like you to take a look.

I think a better syntax for this would be to add a new method called `connection`, that let's you specify with which connection you want to work with, instead of adding this feature to the default consumer command only.

Something like this:

```php
Kafka::connection('your-connection-name')->publishOn(...)
```

**Comment by SergkeiM** [comment]
Hi @stefanovik @mateusjunges 

Added a PR for requested changes by @mateusjunges https://github.com/stefanovik/laravel-kafka/pull/1/files

**Comment by stefanovik** [comment]
HI @SergkeiM, looking at them right now

**Comment by mateusjunges** [comment]
Closing this as I don't think this is the best implementation for this feature. I'll think about it a bit and come up with a better API. Thanks for taking the time to implement it tho ❤️

**Pull request closed without merging**